### PR TITLE
adding note about ignoring sessionToken

### PR DESCRIPTION
**Pull request by chrisdmills-okta**
## Description:
- **What's changed?** Andrea from dev sup requested that we add this note, as customers frequently run into this issue and it results in a significant number of support tickets. I ended up adding a bit more general context near the top, with the note underneath, as it could potentially affect any of the scenarios covered in the body of the guide.
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-477991](https://oktainc.atlassian.net/browse/OKTA-477991)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/session-cookie/main/index.md [hunk 1]
```diff
@@ -24,7 +24,9 @@ Okta uses an HTTP session cookie to provide access to your Okta org and apps acr
 
 > **Note:** Some browsers have begun blocking third-party cookies by default, which disrupts Okta functionality in certain flows. See [FAQ: How Blocking Third Party Cookies Can Potentially Impact Your Okta Environment](https://support.okta.com/help/s/article/FAQ-How-Blocking-Third-Party-Cookies-Can-Potentially-Impact-Your-Okta-Environment).
 
-Okta sessions are created and managed with the [Session API](/docs/reference/api/sessions/).
+Okta sessions are created and managed with the [Session API](/docs/reference/api/sessions/). A session token is sent as part of a request, contained in a `sessionToken` parameter. If the request is successful, the session cookie is set by a `Set-Cookie` header in the response.
+
+> **Important**: By default, Okta Classic orgs ignore the `sessionToken` in a request if there is already a session cookie set in the browser. It is possible to flip this behavior in a specific org by setting an internal flag; talk to your Okta rep to find out more.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/session-cookie/main/index.md:29]:
```suggestion
> **Important**: By default, Okta Classic orgs ignore the `sessionToken` in a request if there is already a session cookie set in the browser. If required, you can change this default behavior for a specific org. Contact your Okta rep for more information.
```

Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/session-cookie/main/index.md:29]:
```suggestion
> **Important**: By default, Okta Classic orgs ignore the `sessionToken` in a request if there is already a session cookie set in the browser. If required, you can change this default behavior for a specific org. Contact your Okta rep for more information.
```

Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/session-cookie/main/index.md:29]: Suggestion for the last sentence. There was some talk of standardizing on a "contact your Okta rep" terminology, but not sure we decided on it yet. Approving if this is something we can work out later.

**Review by brianduffield-okta — APPROVED** [review]

**Comment by chrisdmills-okta** [comment]
Thanks @brianduffield-okta ; agree that standard wording would be good.

**Pull request merged**